### PR TITLE
Added 8 more English quotes

### PR DESCRIPTION
**Pull request by holyhearted**

### CODE REVIEW

**Comment by devkennyy** [comment]
21 commits for 8 quotes?

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -35760,6 +35760,54 @@
...
       "id": 6114
+    },
+    {
+      "text": "\"Wild women don't get the blues\", but I find that lately I've been crying like a tall child.",
+      "source": "First Love / Late Spring",
+      "length": 92,
+      "id": 6115
+    },
+    {
+      "text": "I drive when it rains; at night when it rains, I drive. And the headlight spirits, they lead me down the Styx- so black it shines.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:35771]: Missing space before `-`

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -35760,6 +35760,54 @@
...
+      "source": "Carry Me Out",
+      "length": 130,
+      "id": 6116
+    },
+    {
+      "text": "I never said I'd be alright, just thought I could hold myself together. But I couldn't breathe, I went outside- don't know why I thought it'd be any better.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:35777]: Same thing here

**Comment by holyhearted** [comment]
should be good now

**Review by Miodec — APPROVED** [review]

**Pull request merged**